OpenSim/Examples/Plugins/AnalysisPluginExample/AnalysisPlugin_Template.cpp

```diff
@@ -235,8 +235,11 @@ record(const SimTK::State& s)
         SimTK::Vec3 com = body.getMassCenter();
 
         // GET POSITIONS AND EULER ANGLES
-        _model->getSimbodyEngine().getPosition(s,body,com,vec);
-        _model->getSimbodyEngine().getDirectionCosines(s,body,dirCos);
+        vec = body.getPositionInGround(s);
+        auto dirCosR = body.getTransformInGround(s).R();
+        for(auto r = 0u; r < 3; ++r)
+            for(auto c = 0u; c < 3; ++c)
+                dirCos[r][c] = dirCosR[r][c];
```

Comment:
Can L239–244 be done more concisely using [`Rotation::convertThreeAxesRotationToThreeAngles()`](https://simtk.org/api_docs/simbody/3.5/singletonSimTK_1_1Rotation__.html#a8fedf3e4f467b4c46cb3d75c72d1cbe7)? Something like
```cpp
angVec = body.getTransformInGround(s).R()
         .convertThreeAxesRotationToThreeAngles(
         SimTK::BodyRotationSequence,
         SimTK::XAxis, SimTK::YAxis, SimTK::ZAxis);
```
It would be good if the examples made the best use of the API.